client/core/trade.go

```diff
@@ -2609,6 +2609,21 @@ func (c *Core) sendInitAsync(t *trackedTrade, match *matchTracker, coinID, contr
 					},
 				})
 			}
+			if msgErr.Code == msgjson.RPCUnknownMatch {
```

Comment:
Ah, maybe error is still nil if the above errors.As failed

Reply:
> Ah, maybe error is still nil if the above errors.As failed

I had nil one time while testing .. not sure why. and only for init. Does it work if you add back the As test?

Reply:
Yeah should.